templates/credentials/schedule.html

```diff
@@ -3,100 +3,109 @@
+        </h1>
+        <a href="/credentials/your-exams">View your scheduled exams here</a>
+      </div>
+      <div class="col-6">
+        <form method="post">
+          <input type="number"
+                 name="contract_item_id"
+                 value="{{ contract_item_id }}"
+                 hidden />
+          <label class="p-heading--5" for="exam-name">Select your exam</label>
+          <select class="spaced-bottom--smaller is-paper--input"
+                  id="exam-name"
+                  name="name"
+                  required>
+            <option value="linux-essentials">CUE.01 Linux</option>
+          </select>
+          <label class="p-heading--5" for="exam-timezone">Select your timezone</label>
+          <select class="spaced-bottom--smaller is-paper--input"
+                  id="exam-timezone"
+                  name="timezone"
+                  required></select>
+          <label class="p-heading--5 u-no-margin" for="exam-date">Select your preferred day</label>
+          <p>You can schedule your exam to begin {{ time_delay }} from now or later.</p>
+          <input class="spaced-bottom--smaller is-paper--input"
+                 type="date"
+                 id="exam-date"
+                 name="date"
+                 value="{{ date }}"
+                 min="{{ min_date }}"
+                 max="{{ max_date }}"
+                 required />
+          <label class="p-heading--5 u-no-margin" for="exam-time">Select your preferred time</label>
+          <p>
+            Your exam will take up to one hour to complete. Please make sure to plan accordingly before and after to
+            ensure the best exam experience.
+          </p>
+          <input class="spaced-bottom--smaller is-paper--input"
+                 type="time"
+                 id="exam-time"
+                 name="time"
+                 value="{{ time }}"
+                 required />
+          {% if uuid %}<input type="text" id="exam-uuid" name="uuid" value="{{ uuid }}" hidden />{% endif %}
+          <div class="u-align--right">
+            <button class="p-button--positive u-align--right" type="submit" name="submit">Schedule Exam</button>
+          </div>
+        </form>
+      </div>
+    </div>
+  </section>
 
-    <script>
-      {% if timezone and timezone != "" %}
-      const timezone = "{{ timezone }}";
-      {% else %}
-      const timezone = Intl.DateTimeFormat().resolvedOptions().timeZone;
-      {% endif %}
-      const timezoneInput = document.querySelector("#exam-timezone");
-      const allTimezones = Intl.supportedValuesOf('timeZone');
-      allTimezones.map((tz) => {
-        optionElement = document.createElement("option");
-        optionElement.value = tz;
-        optionElement.innerText = tz;
-        if (tz == timezone) {
-          optionElement.setAttribute("selected", true);
-        }
-        timezoneInput.appendChild(optionElement);
-      })
-    </script>
+  <script>
+    {
```

Comment:
Formatting doesn't seem to work well inside `<script>`…  It tries to format it as JS instead of jinja.